TWT/templates/timathon/codejam_listView.html

```diff
@@ -18,6 +18,9 @@ <h1 class="text-center" style="color: rgb(255,255,255);font-family: 'Lovelo Blac
                 <h4 class="card-title" style="text-align: center;">{{ challenge.challenge.title }}</h4>
                 <h6 class="text-muted card-subtitle mb-2" style="text-align: center;">{{challenge.challenge.created_at|date:"Y" }}</h6>
                 <p class="card-text" style="text-align: center;">{{ challenge.challenge.short_desc }}</p>
+                {% if challenge.challenge.youtube_video_link %}
+                    <h4 class="card-text" style="text-align: center;font-size: 18px;">Youtube Video Link: <a href="{{ challenge.challenge.youtube_video_link }}" target="_blank"></a></h4>
```

Comment:
This would be better if it were a button

Reply:
ok so shall i add a button?

Reply:
ur choice. but add custom css to make it look like a button

Reply:
i have made it as a button. see the next commit. i have used to info bootstrap button :mhm: @Shubhaankar-Sharma